bin/GameIndex.yaml

```diff
@@ -34303,7 +34303,7 @@ SLUS-20821:
   region: "NTSC-U"
   compat: 5
 SLUS-20822:
-  name: "Galactic Wrestling"
+  name: "Galactic Wrestling: The Kinnikuman Legacy"
```

Comment:
The actual name is "Galactic Wrestling: Featuring Ultimate Muscle - The Kinnikuman Legacy"
Your choice on this one ;)

https://i109.fastpic.ru/big/2019/0320/33/_59492eafcc2ecfce6cdcf975ab804833.jpg?noht=1&tk=d301&md5=ZgUe7otgsKweMLQD2wt9og&expires=1613930400